site-src/guides/migrating-from-ingress.md

```diff
@@ -0,0 +1,261 @@
+# Migrating from Ingress
+
+The Gateway API is not backward compatible with the [Ingress API](https://kubernetes.io/docs/concepts/services-networking/ingress/). As a result, if you are an Ingress user and would like to migrate, you will need to convert your Ingress resources to Gateway API resources. 
+
+This guide will help you with the conversion. It will:
+
+* Explain the key differences between the Ingress API and the Gateway API.
```

Comment:
Maybe it's also worth adding a section for why you may want to switch to Gateway API.